collectRatesData.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+
+const config = require('./config');
+const helpers = require('./helpers');
+
+const logger = helpers.logger
+
+const runLogic = () => {
+    logger.info('✋ Init the collect rates data')
+
+    helpers.writeDataFileFromJson(config.rates)
+
+    logger.info('🔆 Starting to collect rates data')
+
+    const collectWorker = async () => {
+        const dataAsJson = helpers.readDataFileAsJson()
```

Comment:
You can shorten `dataAsJson` to just `data`.
First because it's not a json any more it's an object.
Second because it's obvious it's some kind of data structure, and json doesn't help understand which structure

Reply:
done